File: LV0/홀짝에 따라 다른 값 반환하기.js
function solution(n) {
  let answer = 0;
  if (n % 2 !== 0) {
    for (let i = 0; i < n + 1; i++) {
      if (i % 2 !== 0 || i === 1) {
        answer += i;
      }
    }
  } else {
    for (let i = 0; i < n + 1; i++) {
      if (i % 2 === 0) {
        answer += i * i;
      }
    }
  }

  return answer;
}

File: LV1/홀짝에 따라 다른 값 반환하기.go
func solution(n int) int {
    answer := 0
    for i := 0; i <= n; i++ {
        if n % 2 != 0 {
            if i % 2 != 0 {
                answer += i
            }
        } else {
            if i % 2 == 0 {
                answer += i * i
            }
        }
    }
    return answer
}